src/makeRobotAccountant.js

```diff
@@ -5,8 +5,20 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let counter = 0;
+
+  const getSum = (num1) => {
+    return (num2) => {
+      counter++;
+
+      return (counter > 3 && counter % 2 === 0)
+        ? 'Bzzz... Error!'
+        : num1 + num2;
+    };
+  };
+
+  return getSum;
```

Comment:
You can just immediately return the function

Reply:
Done✅